components/CommentSection.jsx

```diff
@@ -0,0 +1,109 @@
+import styled from "styled-components";
+import { StyledButton } from "@/components/buttons/DefaultButtons";
+import BackSvg from "./BackSvg";
+
+export default function CommentSection({ game, onEditGame }) {
+  const handleAddComment = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const newComment = formData.get("comment").trim();
+
+    if (newComment) {
+      const updatedGame = {
+        ...game,
+        comments: [
+          { id: Date.now(), text: newComment },
+          ...(game.comments || []),
+        ],
+      };
+      onEditGame(updatedGame);
+      event.target.reset();
+    }
+  };
+
+  const handleDeleteComment = (id) => {
+    const updatedGame = {
+      ...game,
+      comments: game.comments.filter((comment) => comment.id !== id),
+    };
+    onEditGame(updatedGame);
+  };
+
+  const gameComments = game.comments || [];
```

Comment:
the same would go for this line of code. If every game already has an empty array from the start, you don't need this check.